File: cdk/java/vpc/src/main/java/com/miqle/stacks/VpcStack.java
package com.miqle.stacks;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.beans.factory.annotation.Qualifier;
import org.springframework.stereotype.Component;
import software.amazon.awscdk.core.Construct;
import software.amazon.awscdk.core.Environment;
import software.amazon.awscdk.core.Stack;
import software.amazon.awscdk.core.StackProps;

@Component
public class VpcStack extends AbstractStack {

    @Autowired
    public VpcStack(@Qualifier("app") Construct app, Environment environment) {
        stack = Stack.Builder.create(app, "vpc-master-stack")
                .env(environment)
                .build();
    }

}

File: cdk/python/vpc/vpc/vpc_stack.py
from aws_cdk import core
from aws_cdk import aws_ec2 as ec2

class VpcStack(core.Stack):

    def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
        super().__init__(scope, id, **kwargs)
        vpc = ec2.Vpc(self, "VPC", cidr="10.1.0.0/16")
        
